examples/crd-client/main.go

```diff
@@ -15,34 +15,51 @@
 package main
 
 import (
-	"fmt"
 	"strings"
+	"time"
 
 	agonesv1 "agones.dev/agones/pkg/apis/agones/v1"
 	"agones.dev/agones/pkg/client/clientset/versioned"
 	"agones.dev/agones/pkg/util/runtime"
+	"github.com/sirupsen/logrus"
 	"github.com/spf13/pflag"
 	"github.com/spf13/viper"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/rest"
 )
 
 const (
-	gameServerImage = "GAMESERVER_IMAGE"
+	gameServerImage      = "GAMESERVER_IMAGE"
+	isHelmTest           = "IS_HELM_TEST"
+	gameserversNamespace = "GAMESERVERS_NAMESPACE"
 
 	defaultImage = "gcr.io/agones-images/udp-server:0.19"
```

Comment:
What was the motivation for hardcoding the image here? You were getting it as an environment variable, and it looks like you are still passing it in from the yaml below. 

Seems like an env/arg value would be more flexible long term?